nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/TrashServiceImpl.java

```diff
@@ -412,9 +418,24 @@ protected void undeleteDocument(CoreSession session, DocumentModel doc) {
     public DocumentModelList getDocuments(DocumentModel currentDoc) {
         CoreSession session = currentDoc.getCoreSession();
         DocumentModelList docs = session.query(String.format(
-                "SELECT * FROM " + "Document WHERE " + "ecm:mixinType != 'HiddenInNavigation' AND "
-                        + "ecm:isCheckedInVersion = 0 AND ecm:isTrashed = 1 AND " + "ecm:parentId = '%s'",
+                TRASHED_QUERY,
                 currentDoc.getId()));
         return docs;
     }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Remove `@inheritDoc` Javadoc, it's implicit. It's only useful if you want to add additional text to the inherited doc.

Reply:
:+1: